src/lib/stepper/stepper.ts

```diff
@@ -123,6 +126,7 @@ export class MatStepper extends _CdkStepper implements AfterContentInit {
   inputs: ['selectedIndex'],
   host: {
     'class': 'mat-stepper-horizontal',
+    '[class.mat-stepper--alternative-label]': 'alternativeLabel',
```

Comment:
nit: I don't think we use the `--` style anywhere else in our code, use `-` for consistency